docker/docker_transport_test.go

```diff
@@ -103,6 +110,25 @@ func TestNewReference(t *testing.T) {
 	assert.Error(t, err)
 }
 
+var ValidReferenceUnknownDigestTestCases = []struct{ input, dockerRef, stringWithinTransport string }{
```

Comment:
```suggestion
var validReferenceUnknownDigestTestCases = []struct{ input, dockerRef, stringWithinTransport string }{
```
please.

Or, alternatively (weakly preferred), add these to the existing `validReferenceTestCases`, and add a new `isUnknownDigest` field — that way we only need one simple loop in the three tests which use these tests case arrays.

Reply:
Fixed